### PR TITLE
Remove CORS circumvention support for now. Shall be reworked later on.

### PR DESCRIPTION
**Pull request by omefire**

### CODE REVIEW

**Comment by omefire** [comment]
@nikhilkh to review.

File: lib/livereload.js [hunk 1]
```diff
@@ -77,11 +77,6 @@ function StartServer() {
         proxyMiddleware = require('proxy-middleware');
```
Comment (nikhilkh) [inline lib/livereload.js]: Does this need to be deleted now? Also, the corresponding entry in package.json

Comment (nikhilkh) [inline lib/livereload.js]: Do we still use ua-parser too?

**Comment by nikhilkh** [comment]
`this.livereloadConfigFile` is not needed any more - should you remove that?

**Comment by omefire** [comment]
Fixed and Merged.

**Pull request closed without merging**